src/main/pg/motor.h

```diff
@@ -48,6 +48,7 @@ typedef struct motorDevConfig_s {
     uint8_t  motorTransportProtocol;
     uint8_t  useDshotBitbang;
     uint8_t  useDshotBitbangedTimer;
+    uint8_t  motorRemap[MAX_SUPPORTED_MOTORS]; // Reindexing motors for "remap motors" feature in Configurator
```

Comment:
Should this be `motorMapping` or `motorOutputMapping`?

Reply:
@mikeller,
would it be OK like that?:
motorMapping in the C code
motor_mapping in CLI
MSP_MOTOR_MAPPING
MSP_SET_MOTOR_MAPPING

> Should this be `motorMapping` or `motorOutputMapping`?

Reply:
I'd go for `motorOutputMapping` and equivalent. The motors are not actually remapped: If you have `resource motor 5 A1`, and then remap motor output 3 to be provided by motor 5 by `motor_output_mapping = 1, 2, 5, 4`, then resource show should give you `A1: motor 5`. This will make debugging resource conflicts easier.

Reply:
![image](https://user-images.githubusercontent.com/2925027/86006553-c0180f00-b9db-11ea-87b4-a0faf43a5430.png)
duscussed